utils/vscode/carbon.code-snippets

```diff
@@ -2,54 +2,67 @@
   "package": {
     "scope": "carbon",
     "prefix": "package",
-    "body": "package $fileNameWithoutExt",
+    "body": "impl library ${1:$fileNameWithoutExt}",
```

Comment:
I think we should still have `package`, especially since that's the name and the prefix for the snippet; we just shouldn't provide a default. We could always have a separate `library` snippet for the case where the user doesn't want to specify a package.

It's probably also better to leave out `impl`, because every library has an API file but not every library has an `impl` file, so leaving it out will be the right guess slightly more often. It would be ideal if we could infer this from the filename (`impl` files should always have a `.impl.carbon` extension), but I don't know if that's feasible.

Finally, the library name should have quotes around it.

```suggestion
    "body": "package $1 library \"${2:$fileNameWithoutExt}\"",
```

Reply:
Ok, I will do it